### PR TITLE
Adding color-scheme CSS property

### PR DESCRIPTION
**Pull request by ollips**
task-391082

Making the same update in the atlas as in docs-ui to support scrollbar theming.

### CODE REVIEW

File: css/src/core/themes.scss [hunk 1]
```diff
@@ -76,5 +76,11 @@ $theme-names: map.keys($themes);
 @each $key, $value in $themes {
 	.theme-#{$key} {
 		@extend %theme-base-styles;
+
+		@if $key == $root-theme {
+			color-scheme: light;
+		} @else {
```
Comment (wibjorn) [inline css/src/core/themes.scss:82]: Would this work if we added a new theme? How might be do this in a way that could allow for flexibility in the future?

Comment (ollips) [reply css/src/core/themes.scss]: Update approach

File: css/src/tokens/schemes.scss [hunk 2]
```diff
@@ -0,0 +1,5 @@
+$schemes: (
```
Comment (wibjorn) [inline css/src/tokens/schemes.scss:1]: I think an explanatory comment here would be helpful here. One saying that this object's keys should match themes and that it's value will set the color-scheme css property for that theme.

**Comment by wibjorn** [comment]
Needs changeset?

File: css/src/tokens/schemes.scss (hunk 2 above)
Comment (ollips) [reply css/src/tokens/schemes.scss:4]: Updated

**Review by wibjorn — APPROVED** [review]

**Pull request merged**